ui/pages/confirmations/components/confirm/info/hooks/getGasFeesEstimate.ts

```diff
@@ -0,0 +1,46 @@
+import { GasFeeEstimates } from '@metamask/gas-fee-controller';
+import { TransactionMeta } from '@metamask/transaction-controller';
+import { Hex } from '@metamask/utils';
+import {
+  addHexes,
+  multiplyHexes,
+} from '../../../../../../../shared/modules/conversion.utils';
+import { useGasFeeEstimates } from '../../../../../../hooks/useGasFeeEstimates';
+import { HEX_ZERO } from '../shared/constants';
+
+export function getGasFeeEstimate(
```

Comment:
Since this is still a hook, I think best practice is to name them all `useX`, perhaps `useTransactionGasFeeEstimate` to differentiate it from `useGasFeeEstimates`?